### PR TITLE
[CBRD-24399] setup.sh - A shell script for users installing CUBRID with tar distribution.

### PR DESCRIPTION
**Pull request by kisoo-han**
http://jira.cubrid.org/browse/CBRD-24399

**Purpose**
- All CUBRID Distribution (shell, rpm, tar) will contain following directories
  * $CUBRID/tmp
  * $CUBRID/var/CUBRID_SOCK
  * $CUBRID/databases
  * $CUBRID/databases/databases.txt      # size 0 file

- and will contain following setup scripts
  * $CUBRID/share/scripts/setup.sh
to help user for creation of .cubrid.sh, modify .bash_profile, etc.

- CUBRID Distribution will contain additional READMEs
  * **$CUBRID/README_FOR_TAR_INSTALL** (newly added for tar users)
   How to run $CUBRID/share/scripts/setup.sh for tar distribution
-------------------------------------------------------------------------------------------------------------
- READMEs newly added
  * $CUBRID/tmp/**README.txt**
    directory description
  * $CUBRID/var/CUBRID_SOCK/**README.txt**
    directory description

**Implementation**
N/A

Remarks

### CODE REVIEW

File: contrib/readme/README_TAR_INSTALL [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+IGNORE THIS IF YOU INSTALLED CUBRID FROM THE SHELL OR RPM DISTRIBUTION.
+
+------------------
+If you try to install CUBRID from a tar distribution, setup.sh can help you
+to set environment for normal CUBRID operations.
+
+Steps:
+1. Go to the directory where you untared (probably you already there)
+2. run setup.sh
```
Comment (mhoh3963) [inline contrib/readme/README_TAR_INSTALL]: How about changing as "run setup.sh [cubrid_home]"
and removing "or run setup.sh cubrid_home" ?

File: contrib/scripts/setup.sh [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  fi
+else
+  echo "${cubrid_home}/demo/make_cubrid_demo.sh : No such file"
+fi
+
+echo ""
+echo "If you want to use CUBRID, run the following command to set required environment variables."
+if [ $CUBRID_SH_INSTALLED -eq 0 ];then
+        echo "(or you can add the command into your current shell profile file to set permanently)"
+fi
```
Comment (mhoh3963) [inline contrib/scripts/setup.sh:178]: If CUBRID_SH_INSTALLED is 0, I think that it is better to exit after printing an error message.

File: contrib/scripts/setup.sh [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+if [ $# -ne 0 ];then
+  curr=$(pwd)
+  cd $1
+
+  if [ $? -ne 0 ];then
+    echo "$1: no such directory or pemission denied"
```
Comment (airnet73) [inline contrib/scripts/setup.sh]: typo, pemission  to permission

**Review by airnet73 — APPROVED** [review]

File: contrib/readme/README_TAR_INSTALL (hunk 1 above)
Comment (kisoo-han) [reply contrib/readme/README_TAR_INSTALL]: It looks better. Thank you for your comment!

File: contrib/scripts/setup.sh (hunk 2 above)
Comment (kisoo-han) [reply contrib/scripts/setup.sh:178]: Ok, I will change it.

File: contrib/scripts/setup.sh (hunk 3 above)
Comment (kisoo-han) [reply contrib/scripts/setup.sh]: Thank you for your comment!
Fixed

**Review by mhoh3963 — APPROVED** [review]

**Pull request merged**